adapter/slack.py

```diff
@@ -178,7 +178,16 @@ def reaction_added(self, event, context):
             self.negative_reaction(event, client)
 
         return Response(status_code=200)
+    
+    def reaction_removed(self, event):
+        reaction = event["reaction"]
+
+        # Handle negative reactions
+        if reaction == "-1":
+            self.remove_reaction(event, Reaction.NEGATIVE)

```

Comment:
Hey, we just merged the feature for positive reaction. We should also handle it here later, either pull the main and handle it in this PR or later in another PR.

Reply:
I think it's better handled in a different PR to keep this one small